### PR TITLE
Multivariate linear polinomial

### PR DESCRIPTION
**Pull request by arthurpaulino**

### CODE REVIEW

File: YatimaStdLib/MultilinearPolynomial.lean [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+/-- 4x₂x₃ + 12x₂ + 5 -/
+def pol3 := ofSummandsL [(12, [2]), (4, [2, 3]), (5, [])]
+
+/-- 12x₀x₂x₃x₄ + 36x₀x₂x₄ + 15x₀x₄ + 8x₁x₂x₃ + 16x₂x₃ + 24x₁x₂ + 48x₂ + 10x₁ + 20 -/
+def pol1MulPol3 := ofSummandsL [
+  (12, [0, 2, 3, 4]), (36, [0, 2, 4]), (15, [0, 4]),
+  (8, [1, 2, 3]), (24, [1, 2]), (10, [1]),
+  (16, [2, 3]), (48, [2]), (20, [])]
+
+#lspec
```
Comment (mpenciak) [inline YatimaStdLib/MultilinearPolynomial.lean:154]: Maybe add more tests to check for edge cases? Weird things come up all the time with basically anything having to do with the zero polynomial
